src/ape_console/_cli.py

```diff
@@ -123,7 +123,7 @@ def console(project=None, verbose=None, extra_locals=None):
     if console_extras:
         namespace.update(console_extras)
 
-    from traitlets.config import Config
+    from traitlets.config import Config  # type: ignore
```

Comment:
```suggestion
    from traitlets.config.loader import Config
```

I don't think the `type: ignore` is needed as this library publishes typing. However, I think we need to import this class from the correct spot; that is perhaps why Mypy is complaining.